From 7cf976f01ae97568dcb92f709d787ee81006fa82 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Alex=20R=C3=B8nne=20Petersen?= <alex@alexrp.com>
Date: Wed, 29 Oct 2025 07:48:12 +0100
Subject: [PATCH 022/133] std.Target.aarch64: updates for LLVM 21.1.5
X-Developer-Signature: v=1; a=openpgp-sha256; l=659; i=xtex@astrafall.org;
 h=from:subject; bh=V8FueFgBfd3UtA9RdXIBbGf+W0uY6wBG+BLt1wG4AyA=;
 b=owGbwMvMwCW2U4Ij7wZL9ETG02pJDJn7cn9bmn8w5dAzV3LrK9Bf1fBtxSy5851r7wjsef7Be
 lGdQ/jTjoksDGJcDJZiiixFhg3erDrp/KLLymVh5rAygQyRFmlgYGBgYGHgy03MKzXSMdIz1TbU
 MzTUMdIxYuDiFICpPnaGkWGL88ZYE7042Sa7X9snfkxf+1y/9MHFexMmC7xvEgn7c6eR4Z+luip
 vg0bFbS93tq0aZlvF+VxEtX24/GsijjGHhScv5gIA
X-Developer-Key: i=xtex@astrafall.org; a=openpgp;
 fpr=7231804B052C670F15A6771DB918086ED8045B91

(cherry picked from commit 7881a60f1a3c3132fed8b8f8dab4ca7ed73c0edb)
---
 lib/std/Target/aarch64.zig | 2 --
 1 file changed, 2 deletions(-)

diff --git a/lib/std/Target/aarch64.zig b/lib/std/Target/aarch64.zig
index a899531fbf2a..531121157c1d 100644
--- a/lib/std/Target/aarch64.zig
+++ b/lib/std/Target/aarch64.zig
@@ -1714,10 +1714,8 @@ pub const all_features = blk: {
         .description = "Support ARM v9.6a architecture",
         .dependencies = featureSet(&[_]Feature{
             .cmpbr,
-            .fprcvt,
             .lsui,
             .occmo,
-            .sve2p2,
             .v9_5a,
         }),
     };
-- 
2.52.0

